frontend/src/components/search/utils.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Button, Collapse } from "reactstrap";
+import { ArrowToggleIcon, CopyToClipboardButton } from "@certego/certego-ui";
+
+export function ErrorsCollapse({ errors }) {
```

Comment:
I think you created a component like this for the plugin section, do you think is it possible to refactor the code ?